4_arm.ipynb

```diff
@@ -606,6 +1096,19 @@
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Spawn the can on the desk and write a code to pick the can.</p>\n",
```

Comment:
Up to now, I've seen no mention of a can, so I am assuming that we are not talking about any particular can, yet.
So, I changed it to "a can" at the beginning of this sentence.
Also, "a code" was changed to just "code".

    "    <p>Spawn a can on the desk and write code to pick up the can.</p>\n",

Reply:
Thanks, "a can" and "code" sounds appropriate.